pkgs/development/tools/rust/cargo-lambda/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, makeWrapper, cargo-watch }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-lambda";
+  version = "0.11.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-IK4HVj8Y8Vz+mza8G9C+m5JRfNT3BWWdlbQQkJPu6RI=";
+  };
+
+  cargoSha256 = "sha256-oSqoSvv8IiChtduQQA31wItHcsnRBAQgOCrQN4sjcx8=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram $out/bin/cargo-lambda --prefix PATH : ${lib.makeBinPath [ cargo-watch ]}
+  '';
+
+  checkFlags = [
+    "--skip test_download_example"
+    "--skip test_target_dir_from_env"
```

Comment:
Both of these tests fail. The first accesses the network. Nix's build methodology violates the assumptions of the second.

Reply:
Please add a short comment to each

Reply:
Done.